docs/VOLA_API.md

```diff
@@ -0,0 +1,59 @@
+Api to communicate with Vola
+----------------------------
+----------------------------
+The purpose of building this API was to provide information about the events
+in VMS to Vola so that their volunteers could participate in them. Presently,
+these fields of events are sent to Vola via the API::
+
+1. Event Name - Title of the event going to be held.
```

Comment:
I am not too much sure about the description of the fields. @may @nida. PTAL.

Reply:
The description for the fields look alright. 
 
Minor comment: 
- "going to be held" is not required for Event Name. 
-  For Event Name - "Title or name of the event"
-  For the sake of my OCD, please "." at the end of description. 


Also we should think about having start time and end time of events in VMS (VOLA has that i guess).